src/odemis/driver/test/andorcam2_test.py

```diff
@@ -24,15 +24,18 @@
 from __future__ import division
 
 import logging
+from odemis import model
```

Comment:
I have the feeling more test cases should have be added to this file.
For example I cannot find any test cases for these functions:

```
GetTemperatureRange
GetTriggerLevelRange
_setStaticSettings
```

But looking at the amount of changes I think there are more functions which might deserve some updates for there test cases.

Reply:
These functions are private. They shouldn't be called from outside, so I prefer to not test them. We test the public interface, which indirectly relies on these private functions.

Reply:
The first of these two aren't private isn't it?